### ISSUE TITLE
meeting 02/09/2024, questions

### ISSUE THREAD

**Issue by RoccoPaolillo** [cfabiolongo/Semas#2]
1. This asks one agent to execute the command [{'to': "worker1"}]], doesn't it? Does it need to be repeated for each agent? Can we address the same inference > reactor to the whole class? Or anyway an iteration without re-writing each line
        +TASK(X, Y) / DUTY(1) >> [show_line("assigning job to worker1"), -DUTY(1), +TASK(X, Y, 1)[{'to': "worker1"}]]
        +TASK(X, Y) / DUTY(2) >> [show_line("assigning job to worker2"), -DUTY(2), +TASK(X, Y, 2)[{'to': "worker2"}]]
        +TASK(X, Y) / DUTY(3) >> [show_line("assigning job to worker3"), -DUTY(3), +TASK(X, Y, 3)[{'to': "worker3"}]]

1. Can agents be differentiated in different types?
2. How to address the KG of each agent?
3. If we can use any other package, python mesa  would be better for abm https://mesa.readthedocs.io/en/latest/overview.html

Future plans: parametrizable KG (.ini) from the csv of the database, and apply inferences as in https://github.com/RoccoPaolillo/Semas/blob/30eab8a225c11f7b1647b7dd3808756c10ac87ef/front_end.py#L44
Basically that is a changing  of  the KG due to action, through the belief triplet added to the KG. And related  abm action and visualization.
